plugins/org.jkiss.dbeaver.erd.ui/src/org/jkiss/dbeaver/erd/ui/notation/crowsfoot/CrowsFootDiagramNotation.java

```diff
@@ -115,6 +114,24 @@ private void createTargetDecorator(PolylineConnection conn, Color bckColor, Colo
         conn.setTargetDecoration(targetDecor);
     }
 
```

Comment:
Too complicated and too slow (no need to create array list).
Let's create function DBUtils.,isUniqueColumns(columns). It will return true if there a unique index for that columns.

Reply:
Implemented method in `DBUtils.isUniqueColumn`